workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/WorkerBuildData.cs

```diff
@@ -34,7 +36,9 @@ public class WorkerBuildData
                 { BuildTarget.StandaloneWindows, ".exe" },
                 { BuildTarget.StandaloneWindows64, ".exe" },
                 { BuildTarget.StandaloneLinux64, "" },
-                { BuildTarget.StandaloneOSX, "" }
+                { BuildTarget.StandaloneOSX, "" },
+                { BuildTarget.Android, ".apk" },
+                { BuildTarget.iOS, "" }
             };
 
         public WorkerBuildData(string workerType, BuildTarget buildTarget)
```

Comment:
Heads-up, now that #531 is merged, will need to adjust the BuildTargetSupportDictionaryMerges after a rebase/merge too